### PR TITLE
[Translation] Building Your Own Hooks

### PR DESCRIPTION
**Pull request by visshaljagtap**
Hi @arshadkazmi42, please review and let me know if any changes needed!

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
Thanks @visshaljagtap for working on this.
But right now we are prioritizing the pages listed in #1 
Please take pages from there, once those are closed we will start with other pages.
I am parking this PR for now. We will get to this when we start working on content pages

**Comment by visshaljagtap** [comment]
@arshadkazmi42  I have taken the page that is listed in #1 itself also the page is already assigned to me.

**Comment by arshadkazmi42** [comment]
@visshaljagtap Yes. Right.
Sorry about that. This is a valid page. I will start reviewing this soon

**Comment by visshaljagtap** [comment]
Any updates @arshadkazmi42

**Comment by arshadkazmi42** [comment]
@visshaljagtap I have been a bit busy. I will get to it soon

File: content/docs/hooks-custom.md [hunk 1]
```diff
@@ -141,19 +141,20 @@ function FriendListItem(props) {
...
 
-**How does a custom Hook get isolated state?** Each *call* to a Hook gets isolated state. Because we call `useFriendStatus` directly, from React's point of view our component just calls `useState` and `useEffect`. And as we [learned](/docs/hooks-state.html#tip-using-multiple-state-variables) [earlier](/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns), we can call `useState` and `useEffect` many times in one component, and they will be completely independent.
+**एक कस्टम Hook को अलग-थलग अवस्था कैसे मिलती है?** प्रत्येक Hook *call* में एक पूरी तरह से अलग state है | क्योंकि हम`useFriendStatus` सीधे कॉल करें, React के दृष्टिकोण से हमारे कौम्पोनॅन्ट सिर्फ `useState` and `useEffect` को कॉल करते हैं। और जैसे हमने[सीखा](/docs/hooks-state.html#tip-using-multiple-state-variables) [पहले](/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns), हम एक कौम्पोनॅन्ट में कई बार `useState` and `useEffect` कॉल कर सकते हैं, और वे पूरी तरह से स्वतंत्र होंगे।
 
-### Tip: Pass Information Between Hooks {#tip-pass-information-between-hooks}
+### टिप: Hook के बीच जानकारी पास करें
+ {#tip-pass-information-between-hooks}
```
Comment (arshadkazmi42) [inline content/docs/hooks-custom.md:153]: move this to line 152

File: content/docs/hooks-custom.md [hunk 2]
```diff
@@ -141,19 +141,20 @@ function FriendListItem(props) {
...
 
-Since Hooks are functions, we can pass information between them.
+चूंकि Hook फंक्शन हैं, हम उनके बीच जानकारी दे सकते हैं।
 
-To illustrate this, we'll use another component from our hypothetical chat example. This is a chat message recipient picker that displays whether the currently selected friend is online:
+इसे समझने के लिए, हम अपने काल्पनिक चैट उदाहरण से एक और कौम्पोनॅन्ट का उपयोग करेंगे | यह एक चैट संदेश प्राप्तकर्ता पिकर है जो प्रदर्शित करता है कि वर्तमान में चुना गया दोस्त ऑनलाइन है:
```
Comment (arshadkazmi42) [inline content/docs/hooks-custom.md:157]: `करेंगे |` => `करेंगे।`

File: content/docs/hooks-custom.md [hunk 3]
```diff
@@ -183,25 +184,24 @@ function ChatRecipientPicker() {
   );
 }
 ```
+हम चुना गया friend ID को `recipientID` state variable में रखते हैं, और इसे अपडेट करेंगे यदि यूजर `<select>` पिकर में एक अलग दोस्त चुनता है।
```
Comment (arshadkazmi42) [inline content/docs/hooks-custom.md:187]: `चुना गया` => `चुने गए`

Comment (arshadkazmi42) [inline content/docs/hooks-custom.md:187]: `variable` => `वेरिएबल`

File: content/docs/hooks-custom.md [hunk 4]
```diff
@@ -183,25 +184,24 @@ function ChatRecipientPicker() {
...
-Because the `useState` Hook call gives us the latest value of the `recipientID` state variable, we can pass it to our custom `useFriendStatus` Hook as an argument:
+क्योंकि `useState` Hook कॉल हमें` recipientID` state variable का नवीनतम वैल्यू देता है, इसीलिए हम इसे अपने कस्टम `useFriendStatus` हुक में एक आर्गुमेंट के रूप में पास कर सकते हैं:
 
 ```js
   const [recipientID, setRecipientID] = useState(1);
   const isRecipientOnline = useFriendStatus(recipientID);
 ```
 
-This lets us know whether the *currently selected* friend is online. If we pick a different friend and update the `recipientID` state variable, our `useFriendStatus` Hook will unsubscribe from the previously selected friend, and subscribe to the status of the newly selected one.
+इससे हमें पता चलता है कि *वर्तमान में चयनित* मित्र ऑनलाइन है या नहीं| अगर हम एक अलग दोस्त चुनें और `recipientID` state variable अद्यतन करें, तो हमारा `useFriendStatus` Hook पहले से चयनित दोस्त से सदस्यता समाप्त कर देगा, और नव चयनित एक की स्टेटस के लिए सदस्यता लें।
```
Comment (arshadkazmi42) [inline content/docs/hooks-custom.md:196]: `नहीं|` => `नहीं।`

**Comment by visshaljagtap** [comment]
@arshadkazmi42 Done with the mentioned changes please review!

**Comment by visshaljagtap** [comment]
> @arshadkazmi42 Done with the mentioned changes please review!

any updates @arshadkazmi42

**Comment by visshaljagtap** [comment]
Awaiting @arshadkazmi42

**Comment by arshadkazmi42** [comment]
@visshaljagtap little occupied. will get to it soon

**Comment by arshadkazmi42** [comment]
We are moving our focus to beta pages. So closing this now. 

Sorry about this, but feel free to pick a new page from #168

**Pull request closed without merging**